scripts/update_proto_annotations.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add a comment what it does and when to use it.

Reply:
Done